src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/CourtCaseEventsServiceTest.kt

```diff
@@ -189,25 +188,4 @@ class CourtCaseEventsServiceTest {
     verify(offenderService).processAssociatedOffenders(personEntity, person)
     verify(prisonerService).processAssociatedPrisoners(personEntity, person)
   }
-
-  @Test
-  fun `should not create new defendant when multiple existing records are found`() {
-    // Given
-    val pncNumber = "PNC12345"
-
-    val person = Person(
-      familyName = "Jones",
-      givenName = "Billy",
-      dateOfBirth = LocalDate.of(1969, 8, 15),
-      otherIdentifiers = OtherIdentifiers(pncIdentifier = PNCIdentifier.from(pncNumber)),
-    )
-
-    // When
-    courtCaseEventsService.processPersonFromCourtCaseEvent(person)
-
-    // Then
-    verifyNoInteractions(personRecordService)
-    verifyNoInteractions(offenderService)
-    verifyNoInteractions(prisonerService)
-  }
```

Comment:
This doesn't test what it says it does. It only passes because the PNC is in an invalid format, and in my view is covered by the earlier test `should call telemetry service when exact match found for Court Case Event`